client/ayon_traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -451,13 +451,23 @@ def _get_data_from_csv(
             product_item.folder_path
             for product_item in product_items_by_name.values()
         }
-        folder_ids_by_path: Dict[str, str] = {
+        folder_ids: Dict[str, str] = {
             folder_entity["path"]: folder_entity["id"]
             for folder_entity in ayon_api.get_folders(
                 project_name, folder_paths=folder_paths, fields={"id", "path"}
             )
         }
-        missing_paths: Set[str] = folder_paths - set(folder_ids_by_path.keys())
+        missing_paths: Set[str] = folder_paths - set(folder_ids.keys())
+        if missing_paths:
+            folder_ids_by_name: Dict[str, str] = {
+                folder_entity["path"]: folder_entity["id"]
+                for folder_entity in ayon_api.get_folders(
+                    project_name,
+                    folder_names=missing_paths, fields={"id", "path"}
+                )
+            }
+            folder_ids.update(folder_ids_by_name)
```

Comment:
I think this is interesting, but I also think we should exclude any that already have `/` inside the folder paths to ensure we disallow implicitly matching another folder by name if a folder path was initially passed.

Also, I think we may be better off being even more explicit and allowing a dedicated folder name column instead of folder path so that if a user uses that it's clear that the column allows matching any existing folder? 

---

Note also that this code is buggy - because below we're also using `folder_ids_by_path` to query the task entities - however those found by folder name are not included in that lookup.

Reply:
The folder path column should be sufficient. Let's not add another column if the information can be extracted from the folder path.